### ISSUE TITLE
Exceptions thrown in the trace filter

### ISSUE THREAD

**Issue by andytill** [andytill/erlyberly#109]

**Comment by andytill** [comment]
This happens when BREAK or NODE DOWN rows are in the table of trace logs.

```java
Exception in thread "JavaFX Application Thread" java.lang.NullPointerException
        at erlyberly.format.ErlangFormatter.modFuncArityToString(ErlangFormatter.java:128)
        at erlyberly.TraceLog.appendModFuncArity(TraceLog.java:179)
        at erlyberly.TraceLog.toCallString(TraceLog.java:149)
        at erlyberly.TraceLog.tracePropsToString(TraceLog.java:116)
        at erlyberly.TraceLog.toString(TraceLog.java:108)
        at erlyberly.DbgTraceView.lambda$onTraceFilterChange$6(DbgTraceView.java:283)
        at javafx.collections.transformation.FilteredList.refilter(FilteredList.java:327)
        at javafx.collections.transformation.FilteredList.access$000(FilteredList.java:50)
        at javafx.collections.transformation.FilteredList$1.invalidated(FilteredList.java:102)
        at javafx.beans.property.ObjectPropertyBase.markInvalid(ObjectPropertyBase.java:111)
        at javafx.beans.property.ObjectPropertyBase.set(ObjectPropertyBase.java:146)
        at javafx.collections.transformation.FilteredList.setPredicate(FilteredList.java:125)
        at erlyberly.DbgTraceView.onTraceFilterChange(DbgTraceView.java:282)
```

**Comment by andytill** [comment]
Fixed in 678daa97f5d7cb78200932d3a347daa4aeaa4aa0
